code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -290,6 +338,21 @@
 	armor = list(MELEE = 35,  BULLET = 15, LASER = 20, ENERGY = 10, BOMB = 50, BIO = 100, RAD = 50, FIRE = 50, ACID = 75, STAMINA = 20)
 	light_range = 6
 	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator)
+	actions_types = list(
+		/datum/action/item_action/toggle_helmet_light,
+		/datum/action/item_action/toggle_beacon_hud/explorer
+		)
+
+/obj/item/clothing/head/helmet/space/hardsuit/exploration/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/team_monitor, EXPLORATION_TRACKING, -1)
```

Comment:
I think itsmeow might have changed how this worked, you'll have to ask them if this is still valid code. @itsmeow

Reply:
Yep, they said to use the team_monitor/worn subtype, changed it to that.